package.json

```diff
@@ -63,10 +63,12 @@
     "aegir": "^10.0.0",
     "chai": "^3.5.0",
     "eslint-plugin-react": "^6.9.0",
+    "go-ipfs-dep": "^0.4.5",
     "gulp": "^3.9.1",
     "hapi": "^16.1.0",
-    "interface-ipfs-core": "~0.24.1",
+    "interface-ipfs-core": "github:ipfs/interface-ipfs-core#fix\/pubsub-tests",
```

Comment:
To clarify, I'm pulling in interface-ipfs-core from a branch to make sure the tests run on CI. Before merging, we need that PR merged and use the regular version for the module. Ie. this is not intended as final.